include/http/libcurl_http_client.h

```diff
@@ -244,6 +244,20 @@ namespace azure {  namespace storage_lite {
             curl_global_cleanup();
         }
 
+        int set_curl_ca_path(const std::string& capath)
+        {
+            for (int i = 0; i < m_size; i++) {
+                CURL *h = m_handles.front();
```

Comment:
Consider the case that handler in release_handle, if h is released after set_curl_ca_path, which means when h is pushed back in to m_handles, the CURLOPT_CAPATH is not set for h.